sandbox/src/mocks/r4/sendQuestionnaireResponse/requests/basicQuestionnaireResponse.json

```diff
@@ -0,0 +1,42 @@
+{
+  "resourceType": "QuestionnaireResponse",
+  "questionnaire": "https://fhir.nhs.uk/Questionnaire/ERS-ShortList",
+  "basedOn":  [
+    {
+      "reference": "ServiceRequest/a.832db7fa-ebdd-44b6-ab3b-8329c2d43149/_history/5"
+    }
+  ],
+  "subject": {
+    "identifier": {
+      "system": "https://fhir.nhs.uk/Id/nhs-number",
+      "value": "9912003888"
+    },
+  },
+  "status" : "completed",
+  "authored" : "2022-01-11T16:40:00+00:00",
+  "item":  [
+    {
+      "linkId": "searchCriteria",
+      "answer":  [
+        {
+          "valueString": "HealthcareService?supportedSpecialty=general-medicine&supportedPriority=routine"
```

Comment:
As per the [Question](https://nhsd-confluence.digital.nhs.uk/pages/viewpage.action?pageId=534350452) we will need to look closer at service search to see if we can do it restfully like this. It would be nice if we could to avoid an operation.